src/Features/Context/TiqrContext.php

```diff
@@ -490,16 +459,14 @@ public function theLogsAre(TableNode $table)
             $logs = array_slice($logs, count($rows));
             Assertion::noContent($logs, var_export($logs, true));
         } catch (AssertionFailedException $exception) {
-            $yml = implode(PHP_EOL, array_map(function ($log) {
-                return sprintf(
-                    '| %s | %s | %s |',
-                    $log[0],
-                    $log[1],
-                    isset($log[2]['sari']) ? 'present' : ''
-                );
-            }, $logs));
-
-            throw new \Exception($exception->getMessage() . PHP_EOL . $yml);
+            $yml = implode(PHP_EOL, array_map(fn($log): string => sprintf(
```

Comment:
I would suggest to use a private descriptive method here and hide the complexity inthere...

Reply:
Not changing this, testcode